setup.py

```diff
@@ -45,7 +50,7 @@ def _get_authors_str():
         'Operating System :: POSIX :: Linux',
         'Programming Language :: Python :: 3 :: Only',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.4',
+        'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
```

Comment:
@kamidzi Is 3.7 considered stable? I'm not sure it would be for all dependencies...